### PR TITLE
[US_ND] Update FTR referrals by LSIR chart

### LINKED ISSUE #804: Add 'No Score' for individuals with no assessment score in ND
**What needs to be done? Why does it need to be done?**
The LSIR referral plot on the ND dashboard currently does not include bars for individuals who have no assessment score. The chart should now include the 'No Score' label for the individuals with `NOT_ASSESSED` as the value for their assessment score. 

**Additional context**
See [this update](https://github.com/Recidiviz/recidiviz-data/issues/5910) in recidiviz-data.

### PR DESCRIPTION
**Pull request by alexabatino**
## Description of the change

The LSIR referral plot on the ND dashboard currently does not include bars for individuals who have no assessment score. This PR updates the chart to now include the 'No Score' bars for those who have `NOT_ASSESSED` as the value for their assessment score.

For more context, see this issue in [recidiviz-data](https://github.com/Recidiviz/recidiviz-data/issues/5910).

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #804

This PR is dependent on the backend deploy for [this PR](https://github.com/Recidiviz/recidiviz-data/pull/5954) in recidiviz-data. 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/components/charts/programming/FtrReferralsByLsir.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ const FtrReferralsByLsir = ({
     lsirScoreBuckets
   );
 
-  const chartLabels = lsirScoreBuckets;
```
Comment (Catacola) [inline src/components/charts/programming/FtrReferralsByLsir.js:126]: Let's make this a global constant like how `lsirScoreBuckets` is. Now that we're changing it from the passed labels, we should move it up to the top so it's super obvious to any future dev where these labels are defined.

**Review by Catacola — APPROVED** [review]

**Comment by jovergaag** [comment]
@alexabatino Are you waiting for a dependency to merge this?

**Comment by alexabatino** [comment]
> @alexabatino Are you waiting for a dependency to merge this?

Ah, sorry I forgot to land this! I was waiting for [this change](https://github.com/Recidiviz/recidiviz-data/pull/5954) to deploy in recidiviz-data, but if I'm not wrong, I'm no longer waiting on this and can land this PR!

**Comment by jovergaag** [comment]
> Ah, sorry I forgot to land this! I was waiting for [this change](https://github.com/Recidiviz/recidiviz-data/pull/5954) to deploy in recidiviz-data, but if I'm not wrong, I'm no longer waiting on this and can land this PR! @jovergaag

No problem, I'm just making my way through PRs bumping the ones that might need bumping :)

**Pull request merged**